spec/models/users_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    before(:example) { get users_path }
+
+    it 'should have correct response status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should include correct placeholder text' do
+      expect(response.body).to include('Users#index')
+    end
+  end
```

Comment:
- [ ] This should make your tests not run because we are supposed to target the text that has been included in the body. however, you included ('Users#index).

HINT: expect(response.body).to include("Welcome to user index page").